packages/reporting-server/rest_server/__dev__.py

```diff
@@ -0,0 +1,19 @@
+import uvicorn
```

Comment:
Why do we need a different startup script for dev?

Reply:
We can use the other one with uvicorn directly.